### ISSUE TITLE
Tagged Content Failed  

### ISSUE THREAD

**Issue by NeelamBisht** [tuespetre/TuesPechkin#187]
Hi,
I am creating a pdf from Html. This is a very basic HTML with Div tag.  My requirement is, it should be ADA compatible. When testing accessibility through Acrobat Pro DC  it is giving me the error  "Tagged Content Failed ". Kindly help to resolve this problem.

**Comment by gsinha** [comment]
I am also facing the same issue.
